generators/wp/templates/chisel-starter-theme/functions.php

```diff
@@ -166,6 +172,26 @@ public function twig_class_name($name = '') {
 		}
 		return implode(' ', $classes);
 	}
+
+	/**
+	 * Verifies existence of the vendor.js file
+	 *
+	 * @return bool
+	 */
+	public function twig_has_vendor () {
+		if ( file_exists( get_template_directory() . '/dist/scripts/vendor.js' ) ) {
+			return true;
+		}
```

Comment:
I think If we visit *.dev while running watch this may misleadingly claim that vendor exists even when it doesn't putting if in else to `if ( ! defined( 'CHISEL_DEV_ENV' )` should solve it.